src/current/v23.1/security-reference/authorization.md

```diff
@@ -134,9 +134,10 @@ Users that own objects cannot be dropped until the [ownership is transferred to
 
 When a user connects to a database, either via the built-in SQL client or a client driver, CockroachDB checks the user and role's privileges for each statement executed. If the user does not have sufficient privileges for a statement, CockroachDB gives an error.
 
+<a id="role-options"></a>
 ### Supported privileges
 
-System-level privileges (also known as global privileges) offer more granular control over a user's actions when working with CockroachDB, compared to the [role options authorization model](#role-options).
+System-level privileges (also known as global privileges) offer granular control over a user's actions when working with CockroachDB.
```

Comment:
i think you can keep the old text, and link to this instead: https://www.cockroachlabs.com/docs/stable/create-role#role-options

it contains all the role options, and is clear about which ones are replaced with system privileges.